### DISCUSSION TITLE
Planned support for qr code login auth?

### DISCUSSION THREAD

**Discussion by niklasgrewe** [supabase/supabase#1139] [Feature Requests]
I am a big fan of QR Code Login. Apps like Discord already use this method to make it easier for users to log in. 

If you already use the mobile app and want to log in to the web app, you can simply scan a QR code with the mobile app and be logged in directly to the web app.

I would be happy if such auth methods are added to Supabase in the future.

**Comment by roo12312** [comment]
Why there are no progress on this much awaited feature?

**Reply by romanr** [reply]
let there be progress!
